src/routes/management/http/json-version-get.ts

```diff
@@ -0,0 +1,61 @@
+import { ServerResponse } from 'http';
```

Comment:
Will have to re-write this with the new class-based syntax, however it should be straightforward. I'd also do the caching of the resonse's here as well since it's more intuitive and remove that logic from the browser/index.ts